routers/user/home.go

```diff
@@ -194,7 +199,19 @@ func Issues(ctx *context.Context) {
 		page = 1
 	}
 
-	repoID := ctx.QueryInt64("repo")
+	// Replace regexp return value with commas, and split
+	repoIDStrings := strings.Split(issueRepoIDsPattern.ReplaceAllString(ctx.Query("repos"), ","), ",")
```

Comment:
Is this a standard method to request an array?

Reply:
I don't know this part of the code is from @oscarlofwenhamn

I'll have a look at this ...

Reply:
Probably not standard, no. I'm afraid it was too long ago since I was working on this, so I don't remember the details of what I was going for that well.
This came about after pagination was introduced to the page in question, which is where I ran into problems. In order to handle the way the pagination operates, I needed to convert the return from `ctx.Query("repos")` into a general format, because I couldn't sort it out in the original format.

Reply:
@oscarlofwenhamn thanks for explaining this ... I rewrited the regex and the look ...